config/config.toml.example

```diff
@@ -37,6 +37,9 @@ mem-quota-query = 34359738368
 # Set to true to enable use of temporary disk for some executors when mem-quota-query is exceeded.
 oom-use-tmp-storage = true
 
+# The location of temporary disk storage using by some executors when mem-quota-query is exceeded.
+tmp-storage-path = "/tmp/tidb/tmp-storage"
```

Comment:
Hi, this default value may break the Windows platform. We may change the default value to `NULL` and use `filepath.Jon(os.TempDir(), "tidb", "tmp-storage")` as default behaviour.